src/main/resources/webapp/js/controllers.js

```diff
@@ -1256,15 +1256,23 @@
            })
         };
 
-        $scope.showRescheduleModal = function(modalHeaderMessage, modalBodyMessage) {
+        $scope.showRescheduleModal = function(modalHeaderMessage) {
```

Comment:
this function is used 2 more time to display error messages, so you can't remove this param